icu4j/main/core/src/test/java/com/ibm/icu/dev/test/CoreTestFmwk.java

```diff
@@ -0,0 +1,85 @@
+// © 2023 and later: Unicode, Inc. and others.
+// License & terms of use: http://www.unicode.org/copyright.html
+
+package com.ibm.icu.dev.test;
+
+import com.ibm.icu.util.ULocale;
+import org.junit.Rule;
+import org.junit.rules.TestName;
+
+/**
+ * A base class for testing within the cross-component tests in the `common_tests` component.
+ *
+ * This subclass of TestFmwk allows the usage of functionality from the `core` component. TestFmwk
+ * is not able to depend on `core` since it is located in `framework`, while `core` tests already
+ * use TestFmwk, which requires that `core` depends on `framework`. We cannot have a cycle in our
+ * dependencies.
+ *
+ * With the allowance for this test class to use functionality from components, we can do things
+ * like assert whether tests are properly resetting the ICU and JDK time zones to their original
+ * values at the end of their execution.
+ */
```

Comment:
nit
```suggestion
/**
 * A base class for testing within the cross-component tests in the `common_tests` component.
 *
 * <p>This subclass of TestFmwk allows the usage of functionality from the `core` component. TestFmwk
 * is not able to depend on `core` since it is located in `framework`, while `core` tests already
 * use TestFmwk, which requires that `core` depends on `framework`. We cannot have a cycle in our
 * dependencies.
 *
 * <p>With the allowance for this test class to use functionality from components, we can do things
 * like assert whether tests are properly resetting the ICU and JDK time zones to their original
 * values at the end of their execution.
 */
```